artifacthub/library/general/poddisruptionbudget/1.0.0/template.yaml

```diff
@@ -5,6 +5,16 @@ metadata:
   annotations:
     metadata.gatekeeper.sh/title: "Pod Disruption Budget"
     metadata.gatekeeper.sh/version: 1.0.0
```

Comment:
@nilekhc would we want to bump the version in this case?

Reply:
I'm happy to bump if desired, however there is no change to the rego/template behavior.

Reply:
Since we are adding a new annotation (that may be used for validation), we should at least bump the patch version IMO. WDYT?

Reply:
Yup. We should bump the patch version.